cva6/sim/cva6.py

```diff
@@ -555,13 +555,13 @@ def run_c(c_test, iss_yaml, isa, target, mabi, gcc_opts, iss_opts, output_dir,
                       linker, gcc_opts, elf))
   cmd += (" -march=%s" % isa)
   cmd += (" -mabi=%s" % mabi)
-  run_cmd(cmd, debug_cmd = debug_cmd)
+  run_cmd_output(cmd.split(), debug_cmd = debug_cmd)
```

Comment:
I think this is why the dashboard is failing. Is it an issue if you restore it?

```suggestion
  run_cmd(cmd, debug_cmd = debug_cmd)
```